AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSLocalDocumentStore.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+#import "MSDBStorage.h"
+#import "MSDataStore.h"
+
+@interface MSLocalDocumentStore : MSDBStorage
```

Comment:
You should think about class design/hierarchy for local document store.
It doesn't seem you need to implement `MSDBStorage` to `MSLocalDocumentStore`. It can just be a property that should be enough for `MSLocalDocumentStore` implementation.
Should you say *local* in the class name? `MSDocumentStore` seems be enough to me.
@clpolet What do you think?